tests/ut/ops/test_expert_load_balancer.py

```diff
@@ -0,0 +1,146 @@
+# fused moe ops test will hit the infer_schema error, we need add the patch
```

Comment:
for ut, I don't think use `assert` is a good way. Please use `unittest` lib instead.

Reply:
And ut should have the same code tree with vllm-ascend. this file should be moved to the right place